### ISSUE TITLE
es-module-shims makes request to browser internal data (blob:...) requiring specific CSP policy

### ISSUE THREAD

**Issue by jean553** [guybedford/es-module-shims#404]
By loading the library, we can see the browser making some requests to cached data stored on the browser memory: `blob:https://our-domain/...`.

This request is made from `es-module-shims.js`.

We have very strict CSP policies. Is this call to cache really necessary and is there a way to disable it ?

**Comment by guybedford** [comment]
Yes, this is necessary, as that is how the polyfill operates in rewriting the specifiers using blob URLs.
